release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -438,6 +438,12 @@ For more information about all-multicast mode, see xref:../networking/hardware_n
 ==== Multi-network policy support for IPv6 networks
 With this update, you can now create multi-network policies for IPv6 networks. For more information, see xref:../networking/multiple_networks/configuring-multi-network-policy.adoc#nw-multi-network-policy-ipv6-support_configuring-multi-network-policy[Supporting multi-network policies in IPv6 networks].
 
+[id="ocp-4-15-CoreDNS-filtration-externalname-service-queries"]
+==== CoreDNS filtration of ExternalName service queries for subdomains
+As of {product-title} {product-version}, CoreDNS has been updated from 1.10.1 to 1.11.1.
+
+This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` Service that shared its name with a top-level domain (e.g. com or org). A query for subdomains of an external service should not resolve to that external service. See the associated CoreDNS Github issue link:https://github.com/coredns/coredns/pull/6162[here] for more information.
```

Comment:
```suggestion
This update to CoreDNS resolved an issue where CoreDNS would incorrectly provide a response to a query for an `ExternalName` Service that shared its name with a top-level domain (e.g. com or org). A query for subdomains of an external service should not resolve to that external service. See the associated link:https://github.com/coredns/coredns/pull/6162[CoreDNS Github issue for more information.
```

Reply:
See the IBM Style Guide for referring to web addresses. https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#referring-to-web-addresses

Reply:
Updated As Suggested.